### ISSUE TITLE
Review sniff handling of PHP 7.3 trailing comma in function calls

### ISSUE THREAD

**Issue by dingo-d** [WordPress/WordPress-Coding-Standards#2040]
Since PHP 7.3, adding a trailing comma in function calls, like:

```php
array_merge( $array_one, $array_two, $array_three, );
```

won't throw a parse error.

This has been motivated by the fact that you could have a trailing comma in an array declaration since always:

```php
array(
  'hello',
  'world',
);
```

and in grouped namespace syntax since PHP 7.2:

```php
use Foo\Bar\{
    Foo,
    Bar,
};
```

There are examples of [what's not allowed](https://wiki.php.net/rfc/trailing-comma-function-calls#what_s_not_allowed) in the RFC.

If I'm not mistaken, `PEAR.Functions.FunctionCallSignature` sniff should handle it, and we are including it in the WordPress-Core ruleset. Once we up the requirement of PHPCS to 3.7.0, we should just add extra test cases to see if it's handled correctly.

Refs:
- PHP RFC: https://wiki.php.net/rfc/trailing-comma-function-calls

Related to #764

**Comment by jrfnl** [comment]
Closing as duplicate of #1363
